src/bot/modules/reminders.ts

```diff
@@ -195,7 +197,8 @@ export default class Reminders {
                 components: [
                   {
                     type: ComponentType.Button,
-                    label: "Plus d'informations",
+                    emoji: '📑',
+                    label: 'Cliquez ici pour recevoir vos propositions !',
```

Comment:
Ce serait plus court, t'en pense quoi ?
```suggestion
                    label: 'Mes propositions',
```